tools/options.py

```diff
@@ -121,3 +121,12 @@ def extract_profile(parser, options, toolchain, fallback="develop"):
                                 " supported by profile {}").format(toolchain,
                                                                    filename))
     return profile
+
+def not_disabled(parser, mcu):
```

Comment:
> I would vote for is_disabled,

The function has to either pass or fail an entire build. I'm willing to go for `is_enabled`

> is_rza1h_disabled()

The function does not test for this. the function test for _all_ Cortex-A support. As it stands, the RZ_A1H is the only cortex-A part we have. Further, it _is_ generic: it tests any mcu you give it to verify that the MCU is enabled in the current release. If we disabled cortex-M7, we would extend this function to disable all cortex-M7 builds as well.

I propose: `mcu_is_enabled`